sparse/src/KokkosSparse_StaticCrsGraph.hpp

```diff
@@ -0,0 +1,569 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef KOKKOSSPARSE_STATICCRSGRAPH_HPP_
+#define KOKKOSSPARSE_STATICCRSGRAPH_HPP_
+
+#include <Kokkos_Core.hpp>
+
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
+
+#if defined(KOKKOS_ENABLE_DEPRECATION_WARNINGS) && defined(KOKKOS_IMPL_DISABLE_DEPRECATED_WARNINGS_PUSH)
+KOKKOS_IMPL_DISABLE_DEPRECATED_WARNINGS_PUSH()
+#endif
+
+#include <Kokkos_StaticCrsGraph.hpp>
+
+namespace KokkosSparse {
+using Kokkos::create_staticcrsgraph;
+using Kokkos::GraphRowViewConst;
+using Kokkos::maximum_entry;
+using Kokkos::StaticCrsGraph;
```

Comment:
I am not sure what to do about `create_mirror[_view]`
I certainly want to pull all overloads from Kokkos into `KokkosSparse::`

Reply:
I don't think Kokkos provides `create_mirror...` functions for any other non-view containers, right? Shall we just be consistent with that and move them into `KokkosSparse` namespace? Kokkos can make the suggestion in the deprecation message.